### PR TITLE
POC: Rough draft of FAQ for maintainers

### LINKED ISSUE #407: Add a FAQ page for new maintainers
Like astropy/astropy.github.com#287 but more focused on maintainers.

For example:

* What is tested on Travis CI and why?
* What is tested on Circle CI and why (as opposed to Travis)?
* What tests are matrixed via Tox as opposed to being build jobs in a CI service and why?
* What does ci-helpers actually do and why does it exist?
* Why is there a distinction between “final tests” and “comprehensive tests”?
* Why did we stop using conda in CI?

### PR DESCRIPTION
**Pull request by pllim**
Fix #407 

**TODO**

- [ ] Collect more questions.
- [ ] Refine the answers.
- [ ] Table of contents?
- [ ] Split into sub-categories?
- [ ] Search box?

### CODE REVIEW

File: faq_maintainers.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+easily support, such as 32-bit architecture, parallel execution, and jobs
+that require access to artifacts.
+</p>
+
+<p>
+<b>Q: What tests are matrixed via tox as opposed to being build jobs in a CI service and why?</b><br/>
+A: Usage of tox was implemented as per
+<a href="https://github.com/astropy/astropy-APEs/blob/master/APE17.rst">APE 17</a>.
+It is used to handle most of the test matrix, except for some of the
+visualization jobs in Circle CI because (Tom R and Stuart M should fill in here).
```
Comment (pllim) [inline faq_maintainers.html:43]: I don't know the answer. 😆

**Comment by pllim** [comment]
@Cadair , there is this error on CircleCI: `unknown exception trying to send webhook to https://giles.cadair.dev/circleci`

**Comment by eteq** [comment]
Might we want to add a short "what happened to astropy-helpers" that just links to the APE?

**Comment by pllim** [comment]
Added one for helpers. Also updated the CI stuff.

**Comment by pllim** [comment]
It has been 4 years and no one contributed to populate this, so I am abandoning this.

**Pull request closed without merging**